src/main/kotlin/no/nav/syfo/behandler/kafka/sykmelding/ReceivedSykmeldingDTO.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.syfo.behandler.kafka
```

Comment:
Får "Package directive does not match the file location" warning på denne og noen av de andre filene i no.nav.syfo.behandler.kafka

Reply:
Ja, kan ha blitt noe krøll etter refaktorering/flytting av eksisterende kode.

Reply:
Tror jeg fikk fikset det nå 👍

Reply:
Ser bra ut nå 👍